pages/index.mdx

```diff
@@ -0,0 +1,224 @@
+  },
+  {
+    label: "Php",
+    icon: phpGray,
+    iconActive: phpColor,
+    href: "https://github.com/Dwolla/dwolla-swagger-php",
+  },
+  {
+    label: "C-Sharp",
+    icon: cSharpGray,
+    iconActive: cSharpColor,
+    href: "https://github.com/Dwolla/dwolla-v2-csharp",
+  },
+  {
+    label: "Kotlin",
+    icon: kotlinGray,
+    iconActive: kotlinColor,
+    href: "https://github.com/Dwolla/dwolla-v2-kotlin",
+  },
+];
+
+<!-- Partners and Integrations LogoLink props -->
+
+export const partnerContent = [
+  {
+    label: "Plaid",
+    icon: plaidGray,
+    iconActive: plaidColor,
+    href: "https://www.dwolla.com/integrations/plaid/",
+  },
+  {
+    label: "Sift",
+    icon: siftGray,
+    iconActive: siftColor,
+    href: "https://www.dwolla.com/integrations/sift/",
+  },
+  {
+    label: "Slack",
+    icon: slackGray,
+    iconActive: slackColor,
+    href: "https://www.dwolla.com/partnerships/slack/",
+  },
+  {
+    label: "Quickbooks",
+    icon: quickbooksGray,
+    iconActive: quickbooksColor,
+    href: "https://www.dwolla.com/partnerships/quickbooks/",
+  },
+  {
+    label: "Apto",
+    icon: aptoGray,
+    iconActive: aptoColor,
+    href: "https://www.dwolla.com/contact/apto-waitlist/",
+  },
+];
+
+export default function HomePage() {
+  return (
+    <>
+      <BannerWrapper>
+        <Banner
+          topic="Integrate with Dwolla and move money smarter."
+          description="From ideation, to implementation, and beyond, the Dwolla API is your connection to the US banking system."
+          button={{
+            text: "Discover the Possibilites",
+            link: {
+              href: "https://accounts-sandbox.dwolla.com/login",
+              external: true,
+            },
+          }}
+        />
+      </BannerWrapper>
+      <ContentContainer>
+        <CardGrid>
+          <Link href="/guides" key="Guides" passHref>
+            <StyledLink>
+              <Card
+                icon={guideIcon}
+                topic="Guides"
+                description="Explore step-by-step walkthroughs to ensure your application is built to best practices."
+              />
+            </StyledLink>
+          </Link>
+          <Link href="/concepts" key="Concepts" passHref>
+            <StyledLink>
+              <Card
+                icon={conceptIcon}
+                topic="Concepts"
+                description="Transform your use case to business and functional requirements."
+              />
+            </StyledLink>
+          </Link>
+          <StyledLink
+            href="https://docs.dwolla.com/"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            <Card
+              icon={apiReferenceIcon}
+              link
+              topic="API Reference"
+              description="View details for all of Dwolla’s API endpoints with a complete reference"
+            />
+          </StyledLink>
+        </CardGrid>
```

Comment:
I think we'll have to stack these vertically at mobile breakpoints somehow. They're pushing off the screen right now

Reply:
D'oh! Just now realizing that I didn't even think about mobile 🤦‍♀️

Reply:
This [commit](https://github.com/Dwolla/dev-portal/pull/73/commits/9a751192eb07d682ba6d686a6a5126ab6017d0e0) should fix that! I ended up using the good old CardGrid.